universal-application-tool-0.0.1/app/services/MessageKey.java

```diff
@@ -53,6 +53,7 @@
   EXTERNAL_LINK("link.externalLink"),
   FILEUPLOAD_VALIDATION_FILE_REQUIRED("validation.fileRequired"),
   FOOTER_SUPPORT_LINK_DESCRIPTION("footer.supportLinkDescription"),
+  LANGUAGE_LABEL("label.language"),
```

Comment:
Suggestion (non-blocking): I've seen the usage of suffix `SR` to denote the string is used specifically for screen reader. I don't feel strongly that convention needs to be followed here though.

Reply:
Oh good idea! I now see Sr used in other places - added!